### ISSUE TITLE
Quarkus2 prints null when started using ant target

### ISSUE THREAD

**Issue by Postremus** [quarkusio/quarkus#18302]
## Describe the bug
When I run a quarkus 2.0.0.Final app using an ant target, I get 2 lines simply stating "null".

This is most likely related to the 

### Expected behavior
When run in a terminal, these 2 nulls are empty lines.

### Actual behavior
```
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2021-07-01 11:42:04,166 INFO  [io.quarkus] (Quarkus Main Thread) ant-null 1.0.0-SNAPSHOT on JVM (powered by Quarkus 2.0.0.Final) started in 2.990s. Listening on: http://localhost:8080
2021-07-01 11:42:04,167 INFO  [io.quarkus] (Quarkus Main Thread) Profile dev activated. Live Coding activated.
2021-07-01 11:42:04,167 INFO  [io.quarkus] (Quarkus Main Thread) Installed features: [cdi, resteasy, smallrye-context-propagation]
2021-07-01 11:42:04,168 INFO  [quarkus] (Quarkus Main Thread) Tests paused, press [r] to resume, [h] for more options>
2021-07-01 11:42:04,168 INFO  [quarkus] (Quarkus Main Thread) null
2021-07-01 11:42:04,168 INFO  [quarkus] (Quarkus Main Thread) null
```

## To Reproduce

Link to a small reproducer (preferably a Maven project if the issue is not Gradle-specific).

Or attach an archive containing the reproducer to the issue.

1. Download the reproducer: 
[ant-null.zip](https://github.com/quarkusio/quarkus/files/6747440/ant-null.zip)

2. IDEA: Open the build.xml file in the editor, right click on the content, add as ant build file
3. You should now have a new view on the right side called "ant"
4. Execute the mvn_quarkus:dev_local target by double clicking it

### Output of `uname -a` or `ver`
MSYS_NT-10.0 NANB7NLNVP2 2.10.0(0.325/5/3) 2018-06-13 23:34 x86_64 Msys

### Output of `java -version`
openjdk version "11.0.7" 2020-04-14 LTS
OpenJDK Runtime Environment Zulu11.39+15-CA (build 11.0.7+10-LTS)
OpenJDK 64-Bit Server VM Zulu11.39+15-CA (build 11.0.7+10-LTS, mixed mode)

### Quarkus version or git rev
2.0.0.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)
Apache Maven 3.8.1 (05c21c65bdfed0f71a2f2ada8b84da59348c4c5d)
Maven home: C:\eclipse\tools\apache-maven\bin\..
Java version: 11.0.7, vendor: Azul Systems, Inc., runtime: C:\eclipse\tools\java\11
Default locale: de_DE, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"

**Comment by junbach** [comment]
It's the same with #18287  . I got the same issue as well

**Comment by famod** [comment]
This is basically a duplicate of #18287, so I'm going to close this.
The problem is fixed in `main` (Quarkus 2.1), but the PR was flagged for backporting so it _might_ end up in 2.0.x.
Thanks for reporting!

**Comment by Postremus** [comment]
Thanks, works!
